package.json

```diff
@@ -7,8 +7,7 @@
   "jsnext:main": "es6/index",
   "types": "types/index.d.ts",
   "sideEffects": [
-    "./*/index.js",
-    "./*/polyfill.js"
+    "./*/index.js"
```

Comment:
Can I remove `index.js` as well ? Not sure if it was added as having side effects because of the polyfill.

Reply:
Yes, I think so.